Engine/csound_standard_types.c

```diff
@@ -501,6 +633,31 @@ void add_standard_types(CSOUND* csound, TYPE_POOL* pool) {
     csoundAddVariableType(csound, pool, (CS_TYPE*)&CS_VAR_TYPE_ARRAY);
     csoundAddVariableType(csound, pool, (CS_TYPE*)&CS_VAR_TYPE_INSTR);
     csoundAddVariableType(csound, pool, (CS_TYPE*)&CS_VAR_TYPE_INSTR_INSTANCE);
+
```

Comment:
Just wondering about these aliases. Could we do without them? It looks like they only differ in type name having : and ; I wonder if there isn't a cleaner solution.